components/server/src/config.ts

```diff
@@ -307,7 +307,12 @@ export namespace ConfigFile {
             }
         }
 
-        WorkspaceClasses.validate(config.workspaceClasses);
+        if (config.workspaceClasses.filter((c) => c.isDefault).length !== 1) {
+            log.error(
+                "Exactly one default workspace class needs to be configured: " +
```

Comment:
Nit: Could be clarified as "Exactly one workspace class should be configured as default: "